modules/rn-ocp-release-notes-scale-and-perf.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: REFERENCE
+[id="rn-ocp-release-notes-scale-and-perf_{context}"]
+= Scalability and performance
+
+[role=”_abstract”]
```

Comment:
This isn't a short description. I think you could put these all into one module, so you wouldn't have to try to do a short description for each category.

Or, if you do keep them separate modules like this, then they would need their own true short description.